playground/DetailsPage.tsx

```diff
@@ -41,6 +32,17 @@ import {
   Thumbnail,
 } from '../src';
 
+import {
+  HomeMajorMonotone,
```

Comment:
How come this is needed? New iconography?

Reply:
Yep we had to update them to make tophatting more inclusive for design. Here's a screenshot
![image](https://screenshot.click/Storybook_2020-08-12_15-01-20.png)